setup.py

```diff
@@ -60,9 +60,7 @@ def build_extension(self, ext, cmake):
             build_args += ['--', '-j']
 
         env = os.environ.copy()
-        env['CXXFLAGS'] = '{} -DVERSION_INFO=\\"{}\\"'.format(
-            env.get('CXXFLAGS', ''),
-            self.distribution.get_version())
+        env['CXXFLAGS'] = env.get('CXXFLAGS', '')
```

Comment:
No. Do you want to remove this line ?

Reply:
if it's not necessary, probably best to remove so as to not be confusing

Reply:
I was wondering if there could be some use cases where it could help people compile morphio. Like maybe passing an external include directory or disabling an error ?

Reply:
sorry, wasn't clear, I mean specifically the `env['CXXFLAGS'] = env.get('CXXFLAGS', '')` line; if CXXFLAGS already exists, it's passed through, but if it doesn't a blank one is created, no?

Reply:
I see. Then, we don't need `env` variable at all it seems.